web/src/settings_components.ts

```diff
@@ -1482,10 +1482,16 @@ export function initialize_disable_button_hint_popover(
 export function enable_opening_typeahead_on_clicking_label($container: JQuery): void {
     const $group_setting_labels = $container.find(".group-setting-label");
     $group_setting_labels.on("click", (e) => {
+        if ($(e.target).is("a.help_link_widget, a.help_link_widget i")) {
+            // Clicking on the "?" icon should just open the link and there is
+            // no need to open the typeahead or focus the input, so we return.
+            return;
```

Comment:
Added this as suggested.

Since, `e.currentTarget` is being used now, there will be no error even if this if block is removed, since now the expected label element will be used below but that would mean user may observer flashing of typeahead before the link is opened, so it is still better to have this.